src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -276,7 +276,7 @@ private void configureShuffleboard() {
    * 'forwards' direction.
    */
   public void zeroGyroscope() {
-    zeroGyroscope(Rotation2d.fromDegrees(0));
+    zeroGyroscope(Rotation2d.fromDegrees(90));
```

Comment:
I see what you did, but why?

Reply:
For auto pathing, 0 degrees points directly to the right instead of forward. This and the other change keeps teleop the same while letting the drivetrain prioritize auto coordinates.